### PR TITLE
SC-6033 Set SEGMENT_DOWNLOAD_TIMEOUT_MINS

### PR DESCRIPTION
**Pull request by tomasz-kaminski-sonarsource**
List of checkpoint to verify/do before merging a PR.

### Dev checklist

* [ ] Write/update tests
* [x] Functional validation
* [ ] Public documentation update
* [ ] Private documentation update
* [x] Clean commits (should start with a ticket number, clear message, no fixup, wip)
* [x] Maximize code coverage as much as possible

---

### Reviewer checklist

* [ ] Code review
* [ ] Functional validation
* [ ] Check commits are clean

---

### Steps to validate this PR

Fill-in how you performed the validation, and how the reviewer can replicate it:
* [x] Checked if our test suite is still working.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -53,6 +53,9 @@ runs:
       id: cache-sonar-tools
       if: inputs.cache-binaries == 'true'
       uses: actions/cache@v3
+      env:
+        # Default value is 60mins. Failing timeout is same as cache miss.
```
Comment (anita-stanisz-sonarsource) [inline action.yml:57]: What do you think about changing the comment into something like:
```
The default value is 60mins. Reaching timeout is treated the same as a cache miss.
```
?

Comment (tomasz-kaminski-sonarsource) [reply action.yml]: Make sense. Updated,

**Review by nawar-hamo-sonarsource — APPROVED** [review]
LGTM!

**Review by anita-stanisz-sonarsource — APPROVED** [review]
Nice finding!

**Pull request merged**